### PR TITLE
Added two factor setup.

### PR DESCRIPTION
**Pull request by marcushanses**
- [x] https://github.com/userwidgets/model/pull/71
- [x] https://github.com/utily/cryptly/pull/59

### CODE REVIEW

**Comment by marcushanses** [comment]
bild

File: src/components/me/two-factor/translation/index.tsx
Comment (AntonCroner) [inline src/components/me/two-factor/translation/index.tsx:1]: either write translations or remove

File: src/components/me/two-factor/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+				this.key.email,
+				{
+					twoFactor: event.detail.user.twoFactor,
+				},
+				event.detail.code
+			)) && (this.recoveryCodes = event.detail.user.twoFactor?.recoveryCodes)
+	}
+	render() {
+		return (
+			<smoothly-summary>
```
Comment (AntonCroner) [inline src/components/me/two-factor/index.tsx:39]: idk if the summary should be built into the 2fa component. Maybe someone else wants to have it under something else? idk tho

**Review by AntonCroner — CHANGES_REQUESTED** [review]

**Comment by marcushanses** [comment]
bild

**Review by AntonCroner — APPROVED** [review]

**Pull request merged**